Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = ["crates/*"]
+resolver = "2"
```

Comment:
Why is this necessary?

Reply:
Rust nightly complains because some workspace members use `edition 2021` but the workspace does not. This fixes this warning (yes, I'm now using nightly because I can't live without CLions test runner integration)